pages/snapshots/[id].js

```diff
@@ -56,13 +56,14 @@ const SnapshotSummary = ({ resources, initialSnapshot, token }) => {
     customerId = customerId.substring(4);
   }
   const residentCoordinates = geoCoordinates(postcode);
+  const INH_URL = process.env.INH_URL
```

Comment:
Does this need adding to the CI pipeline (in `serverless.yml` since it's a runtime variable, I think)?

Happy to help sort it if so

Reply:
yes indeed I put a comment in https://trello.com/c/J0bmhS4m/62-setup-aws-infrastructure-for-new-snapshot-fork

Reply:
I have amended the serverless file now if you can check again pls
`INH_URL: ${ssm:/frontdoor-snapshot/inh-url}`